code/API_definitions/edge-discovery-service.yaml

```
@@ -0,0 +1,1127 @@
+---
+openapi: 3.0.3
+############################################################################
+#                                     API info                             #
+############################################################################
+info:
+  contact:
+    email: sp-edc@lists.camaraproject.org
+  title: Edge Discovery API
+  version: 0.9.2
```

```suggestion
  version: wip
```

 API versioning needs to be aligned with https://wiki.camaraproject.org/display/CAM/API+versioning according with https://github.com/camaraproject/CallForwardingSignal/discussions/27#discussioncomment-9701045